client/src/models/Position.js

```diff
@@ -14,6 +14,28 @@ export const principalPosition = Settings.fields.principal.position
 export const administratorPosition = Settings.fields.administrator.position
 export const superuserPosition = Settings.fields.superuser.position
 
+export class PositionRole {
+  // Static enumerations
+  static MEMBER = new PositionRole("MEMBER", "member")
+  static DEPUTY = new PositionRole("DEPUTY", "deputy")
+  static LEADER = new PositionRole("LEADER", "leader")
+  #value
+  #humanReadable
+
+  constructor(value, dicKey) {
```

Comment:
Prefer to spell it out: `dicKey` → `dictionaryKey`.

Reply:
Applied